eclipse/sw360
# feat(ReleaseUi):Display AssessmentSummary info from CLI xml in Release details page

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
* Conditions to be met in order to display the `Assessment Summary Information` in `Clearing Details` tab of `Release Details` page:
* There should be only `ONE CLI` xml in Release Attachments, and in case of multiple CLI files attached in release then there should be `ONLY ONE ACCEPTED CLI` xml in Release Attachment.
* If the above condition is not met, then `Show Assessment Summary Info` button will not be shown.
* `Assessment Summary` information should be present in CLI xml, else appropriate message will be shown.

Issue: #1577 

### How To Test?

- Create a Release, and upload a `CLI` xml attachment in Release.
- Now You'll see the button `Show Assessment Summary Info`.
- Clicking on this button should load the Assessment summary information in the UI.

### Screenshot:
**Assessment Summary info**
![image](https://user-images.githubusercontent.com/50870174/176627065-b29d4250-2337-4552-aa2b-b479a8fe2703.png)

![image](https://user-images.githubusercontent.com/50870174/176627289-e46bbda0-7c0e-46e4-b647-866f0b41a377.png)


### Checklist
Must:

Signed-off-by: Abdul Kapti <abdul.kapti@siemens-healthineers.com>
